New OSV tests are failing

This bug for @rhythmrx9 

Looks like the new tests we merged today are having trouble, even though they passed in the PR.  Can you look into it?

```
=========================== short test summary info ============================
FAILED test/test_source_osv.py::TestSourceOSV::test_format_data[cve_entries0]
FAILED test/test_source_osv.py::TestSourceOSV::test_format_data[cve_entries1]
========== 2 failed, 380 passed, 321 skipped, 120 warnings in 59.34s ===========
```

--- a/test/test_source_osv.py
+++ b/test/test_source_osv.py
@@ -153,6 +153,7 @@ class TestSourceOSV:
                 "score": "unknown",
                 "CVSS_version": "unknown",
                 "CVSS_vector": "unknown",
+                "last_modified": "2021-09-26T23:33:39.795406Z",
             },
             "affected_data": [],
         },
@@ -164,6 +165,7 @@ class TestSourceOSV:
                 "score": "unknown",
                 "CVSS_version": "unknown",
                 "CVSS_vector": "unknown",
+                "last_modified": "2022-07-21T05:54:26.524614Z",
             },
             "affected_data": [
                 {

